package.json

```diff
@@ -49,7 +49,7 @@
   },
   "dependencies": {
     "babel-runtime": "^6.23.0",
-    "diff-dom": "2.5.1",
+    "diff-dom": "4.2.8",
```

Comment:
Should we add it with `^` sign to allow upgrade for minor version in future in package-lock.json file?